docs/content/en/docs/Coding Milestones/PoC2023q1/outline.md

```diff
@@ -377,6 +414,31 @@ propagation from workload management workspace to mailbox workspace
 does not attempt to manage the resource and namespace definitions that
 are built into the mailbox workspaces.
 
+The above wording is deliberately restrained, for the sake of
+flexibility regarding resources that are defined one way in the edge
+clusters and another way in workload management workspace.  For
+example, the following scenario is allowed.
+
+- Some central team owns an API group and produces some
+  CustomResourceDefinition (CRD) objects that populate that API group.
+- That team derives APIResourceSchemas from those CRDs and a
+  corresponding APIExport of their API group.
+- That team maintains a kcp workspace holding those APIResourceSchemas
+  and that APIExport.
+- Some workload management workspaces have APIBindings to that
+  APIExport, and EdgePlacement objects that (1) select those
+  APIBinding objects for downsync and (2) select objects of kinds
+  defined through those APIBindings for either downsync or upsync.
+- Those resources are built into the edge clusters by pre-deploying
+  the aforementioned CRDs there.
+- Those resources are _not_ built into the mailbox workspaces.  In
+  this case the APIBindings would propagate from workload management
+  workspace to mailbox workspaces but not edge clusters.
+- As a consequence of those propagated APIBindings, the APIExport's
+  view includes all of the objects (in workload management workspaces,
+  in mailbox workspaces, and in any other workspaces where they
+  appear) whose kind is defined through those APIBindings.
+
```

Comment:
Is there a diagram that describe those relations ? It may help to understand the mappings of binding, exports, resources, between the workload WS, mailbox WS, and pCluster.

Reply:
I have not drawn a picture yet.
Is there information missing from the textual description?

Reply:
(1) Drawings almost always help.  I hope the lack of one does not hold up merging this.

(2) The "sync" done by our processes is eventually consistent, which explicitly admits there may be interim periods of inconsistency.

(3) I suspect your concern is consistency that our automated processes do not even attempt to maintain (e.g., between a resource's schema built into edge clusters vs. the schema given for the same resource in the center).  That can be lacking from the start as well as crop up later!  This is an issue I tried to explicitly defer for later consideration.

(4) The scenario outlined here is no longer the one that we intend to use for the Compliance-to-Policy work.  For C2P we intend to drive CRDs from workload management workspace through mailbox workspaces to edge clusters.

Reply:
I am ok with that answer.
/lgtm